### DISCUSSION TITLE
Report

### DISCUSSION THREAD

**Discussion by abbaphy** [community/community#121007] [Discussions]
### Select Topic Area

General

### Body

Dear GitHub Support Team,

I hope this email finds you well. I  am writing to you because I am currently experiencing an issue with my  GitHub account, and I urgently need assistance to regain access.

for the pass couple weeks, I have been unable to log in to my GitHub account with my  usual credentials. Despite multiple attempts, I continue to receive an  error message indicating that my login credentials are incorrect. I am  certain that I am entering the correct username and password, which  leads me to believe that there may be an issue with my account.

I  am deeply concerned about the security of my account, as I fear it may  have been compromised by unauthorized access. I have reviewed my account  settings and attempted to reset my password using the "Forgot  password?" feature, but I have not been successful in regaining access.

In  light of these circumstances, I kindly request your urgent assistance  in resolving this matter and restoring access to my GitHub account.  Additionally, if there have been any unauthorized activities or changes  associated with my account, I would appreciate any information or  guidance on how to address them.

Please find below the necessary details for my GitHub account:

Username: [http://github.com/abubakarharuna10](https://t.co/c7BDimJjMC)

Email associated with the account: 

I  am available to provide any further information or clarification that  may be required to assist with the resolution of this issue. Your prompt  attention to this matter would be greatly appreciated,

Thank you for your understanding and assistance. I look forward to hearing from you soon.

Best regards,

Abba Haruna

**Comment by LiteBrite82** [comment]
Hi @abbaphy 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by LiteBrite82** [reply]
P.s. I edited your personal email out of the original post; I would hate for your account to get flooded with spam. (since we don't do escalations from the discussions boards)
